cgmanifest.json

```
@@ -11973,8 +11973,8 @@
         "type": "other",
         "other": {
           "name": "msopenjdk-11",
-          "version": "11.0.14.1+1-LTS-31207.aarch64",
-          "downloadUrl": "https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/aarch64/msopenjdk-11-11.0.14.1+1-LTS-31207.aarch64.rpm"
+          "version": "11.0.15-1.aarch64.rpm",
```

Is the `.rpm` suffix correct?
```suggestion
          "version": "11.0.15-1.aarch64",
```
```suggestion
          "version": "11.0.15-1.aarch64.rpm",
```

---

good catch, that's a typo